### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Turbo87**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,72 @@
...
 
+
 Installation
 ------------------------------------------------------------------------------
 
 ```
 ember install eslint-config-simplabs
 ```
 
+or if you don't used Ember CLI in your project:
```
Comment (marcoow) [inline README.md]: This sounds like you're referring to legacy non-Ember CLI Ember projects while it actually just refers to non-Ember projects (unless I'm mistaken).

Comment (Turbo87) [reply README.md]: @marcoow well, without Ember CLI there is no `ember` CLI tool you can use to install this 😉

Comment (marcoow) [reply README.md]: Right, saying that this is about non-Ember projects would still be clearer, wouldn't it?

Comment (Turbo87) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -1,15 +1,72 @@
...
 
+
 Installation
 ------------------------------------------------------------------------------
 
 ```
 ember install eslint-config-simplabs
 ```
 
+or if you don't used [Ember.js](http://emberjs.com/) in your project:
```
Comment (marcoow) [inline README.md]: ~used~/use

**Review by marcoow — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Turbo87) [reply README.md]: fixed, thanks

**Review by marcoow — APPROVED** [review]

**Pull request merged**